pages/developers/packaging.md

```diff
@@ -260,10 +260,7 @@ classifiers =
     Operating System :: Unix
     Programming Language :: C++
     Programming Language :: Python
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
```

Comment:
If you remove all these lines you might as well use Python 3 *only*

Reply:
Good point!